src/components/Filters/Brush.js

```diff
@@ -167,22 +153,22 @@ class Brush extends Component {
 
     this.xAxis = axisBottom().scale(updatedScales.xScale)
       .tickFormat((date,i) => {
-        // console.log(timeYear(date))
-        // console.log(i, date)
-        // console.log(timeDay.offset(date, -31))
-        // console.log(" ")
-        if (timeYear(date) < timeDay.offset(date, -31)) {
-          return timeFormat('%b')(date);
-        } else {
-          return timeFormat('%Y')(date);
-        }
+        // if (timeYear(date) < timeDay.offset(date, -1)) {
+          return timeFormat('%b-%d')(date);
```

Comment:
Depending on how often this format is used, you can extract the format string or make a new function that spits out the desired format. This is mostly for readability and reuse-ability. 

`const monthDateFormat = '%b-%d'`